### ISSUE TITLE
Wrong status of subscriptions AWS

### ISSUE THREAD

**Issue by kwodzynski** [servicecatalog/development#625]
Glassfish 3.1.2.2
Java 7u45
delivery V16.1 Fix7

I have two aws subscriptions. In AWS dashboard I can see their status: "Running"
![aws-dash](https://cloud.githubusercontent.com/assets/18006407/24294700/e86fb976-1097-11e7-9307-aefed2bb43e9.PNG)

But in subscription in marketplace (/oscm-portal/marketplace/account/subscriptions.jsf) I can see wrong status of these subscription: "Pending"
![sub](https://cloud.githubusercontent.com/assets/18006407/24294795/4125c0b0-1098-11e7-9f0d-cc39cbf5001c.PNG)

Therefore I cannot assign myself or other user to the subscriptions.
Moreover, if I change state of subscription via aws dashboard (for example to stopped), status in marketplace is still unchanged.

**Comment by SylviaDehn** [comment]
@kwodzynski: 
Your second issue, that the changes from the aws dashboard should be visible in marketplace is invalid.
You have to make state changes from CTMG to see these changes. CTMG does not check aws for status
changes.

For the first issue, I am presuming that the state never changes from pending. Is that correct? Did you check the instance dashboard? What is the status there?
e.g. http://<myserver>:8880/oscm-app/controller/?cid=ess.aws

**Comment by SylviaDehn** [comment]
@kwodzynski : I just retested aws and it works fine with me. Please add your logfile.
Was this a "new" installation for app (i.e. created new domain?) or was it an update installation.
Also check http://:8880/oscm-app-aws if you can log into the controller.

**Comment by kwodzynski** [comment]
@SylviaDehn I created a new subscription and I checked again. The status of the subscription was "pending" but it changed when I opened the link from the email. Sorry, I did not know about it earlier (I mean confirmation e-mail link). I can log in to the aws controller dashboard and the status of my subscription is "COMPLETED" now. FYI, I have the updated app-domain.

I think that we can consider this bug as invalid. Should I close the bug?
